src/webview/git-import/app/gitImport.tsx

```diff
@@ -236,6 +241,11 @@ export class GitImport extends React.Component<DefaultProps, {
         });
     }
 
+    getTrimmedURL = (value: string): string => {
```

Comment:
This function doesn't use the component's state, so you can move it out of the class